support-frontend/assets/components/button/button.scss

```diff
@@ -47,6 +48,26 @@
   }
 }
 
+.component-button--tertiary {
+  color: gu-colour(neutral-20);
+  background-color: transparent;
+  border: solid 1px gu-colour(neutral-20);
+
+  &:hover, &:focus {
+    background-color: gu-colour(neutral-97);
+  }
+}
+
+.component-button--tertiaryFeature {
+  color: #FFE500;
```

Comment:
can we use variables for this,  I think there is one called highlight-main which is a yellow

Reply:
Yes, good pickup.